packages/ui/src/kit/VTabs/VTabs.vue

```diff
@@ -44,7 +44,7 @@ export default {
   },
 
   data: () => ({
-    activeTabIdx: null,
+    activeTabIdx: 0,
```

Comment:
I would to suggest don't use abbreviation like `idx`. In this case, 2 folded letters don't only to give us any readability impact but even reduce it. `activeTabIndex`, because clear and explicit better.

Reply:
maybe by default `activeTabIdx` maybe set not to `0` but to `-1` ?

Reply:
@lamartire but what about `-1` default value?
if there are no any active tabs, first tab always be selected. is it fine?